<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eightPuzzle</class>
 <widget class="QMainWindow" name="eightPuzzle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eightPuzzle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>初始矩阵</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目标矩阵</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="init1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="init9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="final9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭程序</string>
    </property>
   </widget>
   <widget class="QLabel" name="show1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QLabel" name="show9">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>过程展示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>中间过程</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>eightPuzzle</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
